src/rhash/hash.c

```diff
@@ -1918,6 +1918,7 @@ int rc_hash_generate_from_file(char hash[33], int console_id, const char* path)
     case RC_CONSOLE_MEGADUCK:
     case RC_CONSOLE_NEOGEO_POCKET:
     case RC_CONSOLE_ORIC:
+    case RC_CONSOLE_PC_ENGINE:
```

Comment:
This should be moved down into the same block as RC_CONSOLE_NINTENDO, as we also strip the header from PCE ROMs.